jvb/src/main/java/org/jitsi/videobridge/stats/StatsManager.java

```diff
@@ -115,79 +99,16 @@ public void addTransport(StatsTransport transport, long period)
         transports.add(new TransportPeriodicRunnable(transport, period));
     }
 
-    /**
-     * Finds the first instance of {@code Statistics} with a specific runtime
-     * type generated/updated at a specific interval/period.
-     *
-     * @param clazz the runtime type of the {@code Statistics} to be found
-     * @param period the internal/period in milliseconds at which the
-     * {@code Statistics} to be found is generated/updated by this
-     * {@code StatsManager}
-     * @return the first instance of {@code Statistics} with runtime type
-     * {@code clazz} generated/updated every {@code period} milliseconds if any;
-     * otherwise, {@code null}
-     */
-    public <T extends Statistics> T findStatistics(Class<T> clazz, long period)
-    {
-        // XXX The field statistics is a CopyOnWriteArrayList in order to avoid
-        // synchronization here.
-        for (StatisticsPeriodicRunnable spp : statistics)
-        {
-            if (spp.getPeriod() == period && clazz.isInstance(spp.o))
-            {
-                @SuppressWarnings("unchecked")
-                T t = (T) spp.o;
-
-                return t;
-            }
-        }
-        return null;
-    }
-
     /**
      * Gets the (sets of) <tt>Statistics</tt> which this <tt>StatsManager</tt>
```

Comment:
nit: comment is now slightly inaccurate

Reply:
I'm continuing work on this in another PR, will address there